test/tables/sortable-table.test.js

```diff
@@ -90,13 +90,20 @@ test('column can have custom sort function', () => {
 })
 
 test('column can have custom component', () => {
-  const MyCell = () => <td> hi! </td>
+  const MyCell = () => <td> Hi! </td> 
```

Comment:
💯

Reply:
😂 I was gonna use this for the test but then I changed it back (or so I thought)